src/seedu/addressbook/commands/HelpCommand.java

```diff
@@ -21,6 +21,8 @@ public CommandResult execute() {
                 + "\n" + ClearCommand.MESSAGE_USAGE
                 + "\n" + FindCommand.MESSAGE_USAGE
                 + "\n" + ListCommand.MESSAGE_USAGE
+                + "\n" + NameSortCommand.MESSAGE_USAGE
```

Comment:
Following the other command names, is it better to use "SortNameCommand"/"SortEmailCommand"?